# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1315063978 25200
#      Sat Sep 03 08:32:58 2011 -0700
# Node ID a04e32272ecbd0aed8b46a89ff4e21e4d9e8e5dd
# Parent  098c8e6962364d68222b215d76820472c8cafdf1
codesprint: Turn off test warning for various internal helper functions

* __makeinfo__.m, __all_opts__.m, __gnuplot_drawnow__.m, __go_close_all__.m,
__plt_get_axis_arg__.m: Use %!assert(1) to disable test warning.

diff --git a/scripts/help/__makeinfo__.m b/scripts/help/__makeinfo__.m
--- a/scripts/help/__makeinfo__.m
+++ b/scripts/help/__makeinfo__.m
@@ -114,8 +114,11 @@ function [retval, status] = __makeinfo__
 
   unwind_protect_cleanup
     if (exist (name, "file"))
       delete (name);
     endif
   end_unwind_protect
 endfunction
 
+## No test needed for internal helper function.
+%!assert (1)
+
diff --git a/scripts/optimization/__all_opts__.m b/scripts/optimization/__all_opts__.m
--- a/scripts/optimization/__all_opts__.m
+++ b/scripts/optimization/__all_opts__.m
@@ -62,8 +62,12 @@ function names = __all_opts__ (varargin)
       names = names(idx);
     endif
     saved_names = names;
     recursive = false;
   endif
 
 endfunction
 
+
+## No test needed for internal helper function.
+%!assert (1)
+
diff --git a/scripts/plot/__gnuplot_drawnow__.m b/scripts/plot/__gnuplot_drawnow__.m
--- a/scripts/plot/__gnuplot_drawnow__.m
+++ b/scripts/plot/__gnuplot_drawnow__.m
@@ -381,8 +381,12 @@ function retval = have_non_legend_axes (
     all_axes_tags = get (all_axes, "tag");
     legend_axes = strcmp (all_axes_tags, "legend");
     if (! isempty (legend_axes))
       n_legend_axes = sum (legend_axes);
       retval = (n_all_axes - n_legend_axes) > 1;
     endif
   endif
 endfunction
+
+
+## No test needed for internal helper function.
+%!assert (1)
diff --git a/scripts/plot/__go_close_all__.m b/scripts/plot/__go_close_all__.m
--- a/scripts/plot/__go_close_all__.m
+++ b/scripts/plot/__go_close_all__.m
@@ -21,8 +21,12 @@
 ## Undocumented internal function.
 ## @end deftypefn
 
 ## Author: jwe
 
 function __go_close_all__ ()
   close ("all", "hidden");
 endfunction
+
+
+## No test needed for internal helper function.
+%!assert (1)
diff --git a/scripts/plot/__plt_get_axis_arg__.m b/scripts/plot/__plt_get_axis_arg__.m
--- a/scripts/plot/__plt_get_axis_arg__.m
+++ b/scripts/plot/__plt_get_axis_arg__.m
@@ -71,8 +71,12 @@ function [h, varargin, narg] = __plt_get
 
   if (ishandle (h) && strcmp (get (h, "nextplot"), "new"))
     h = axes ();
   endif
 
   narg = length (varargin);
 
 endfunction
+
+
+## No test needed for internal helper function.
+%!assert (1)
